scan/src/pages/DataSourceHomePage.re

```diff
@@ -37,7 +37,18 @@ module Styles = {
 
 [@react.component]
 let make = () => {
-  let dataSourcesOpt = DataSourceHook.getList();
+  let (page, setPage) = React.useState(_ => 1);
+  let limit = 10;
+
+  let dataSourcesCountOpt = DataSourceHook.getList();
```

Comment:
Why we use page=1 and limit =10 as data source count?
*Note: If it is used for temporary. please add TODO